Project.toml

```
@@ -21,17 +21,11 @@ GitHub = "5.1.1"
 HTTP = "0.8"
 JSON = "0.19, 0.20, 0.21"
 RegistryTools = "1.2"
-TimeZones = "0.10"
+TimeZones = "0.10.3, 0.11, 1"
```

I'd rather drop support for pre-1.0 TimeZones.

```suggestion
TimeZones = "1"
```

---

Works for me